### PR TITLE
add clusterview api to list managedclusterset

### PR DESCRIPTION
**Pull request by zhiweiyin318**

### CODE REVIEW

**Review by qiujian16 — COMMENTED** [review]
/lgtm

File: pkg/proxyserver/apis/clusterview/v1alpha1/register.go [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+const GroupName = "clusterview.open-cluster-management.io"
+
+var (
+	GroupVersion  = schema.GroupVersion{Group: GroupName, Version: "v1alpha1"}
+	schemeBuilder = runtime.NewSchemeBuilder(addKnownTypes)
+	// Install is a function which adds this version to a scheme
+	Install = schemeBuilder.AddToScheme
+
+	// SchemeGroupVersion generated code relies on this name
+	// Deprecated
```
Comment (xuezhaojun) [inline pkg/proxyserver/apis/clusterview/v1alpha1/register.go:19]: The function seems to be deprecated since its first day.. is this a typo? @zhiweiyin318

**Pull request merged**